vendors/rakwireless/codecs/test_encode_example.json

```diff
@@ -0,0 +1,13 @@
+[
```

Comment:
This file must be named: `test_encode_rak_standard.json` (as it is related to the `rak_standard.js` codec file).
If there are no tests, you can leave the content to `[]` (empty array of tests).

Reply:
OK, got it.